README.md

```diff
@@ -26,6 +26,23 @@ The `azdev` tool is designed to aid new and experienced developers in contributi
     - For Azure CLI: https://github.com/Azure/azure-cli
     - For Azure CLI Extensions: https://github.com/Azure/azure-cli-extensions
     - Any other repository that you might have access to that contains CLI extensions.
+
+    After forking `azure-cli`, follow the below commands to set up:
+    ```Shell
+    # Clone your forked repository
+    git clone git@github.com:<your-github-name>/azure-cli.git
+    cd azure-cli
+    # Add the Azure/azure-cli repository
+    git remote add upstream git@github.com:Azure/azure-cli.git
+    # Reset the default dev branch to track dev branch of Azure/azure-cli so you can use it to track the latest azure-cli code.
+    git branch dev -u upstream/dev
+    # Develop with a new branch
+    git checkout -b feature_branch
+    ...
+    # When code is ready, push it to your forked repository and submit a PR to merge into the dev branch of Azure/azure-cli repository.
+    git push -u origin feature_branch
```

Comment:
And final step, open pull request, request review, merge.

Reply:
This may be a bit out of scope for this azdev readme which focuses on using azdev to setup, style/linter check and test. A description of all development steps is in the `azure-cli` [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules) doc with the final step as `Submitting Pull Requests`.